packages/server/src/data/db/db-operations.js

```diff
@@ -0,0 +1,13 @@
+const Post = require("../models/post-model");
```

Comment:
are we gonna dump all our db calls here or is this just a temporary thing? ow mayhaps we could make a db-operations module for each collection in our db (e.g. posts, users, etc)

Reply:
mayhaps I'll make db-operations a folder for later